test/integration/setup.ts

```diff
@@ -13,9 +13,6 @@ export async function suiteClientSetup(this: Mocha.Context): Promise<void> {
   await setupClient.bind(this)(serverUrl)
   await ledgerAccept(this.client)
   this.newWallet = generateXAddress({ includeClassicAddress: true })
-  // two times to give time to server to send `ledgerClosed` event
-  // so getLedgerVersion will return right value
-  await ledgerAccept(this.client)
```

Comment:
Not sure what `ledgerAccept` was there for before. Was this related to [ledger_accept](https://xrpl.org/ledger_accept.html) for stand-alone mode, or just a naming coincidence?

Reply:
It's a helper function that basically runs `ledger_accept` in the standalone container (it's in `test/integration/utils.ts`). I guess it was necessary when `getLedgerVersion` listened to a `ledger` subscription.